docker/test/fasttest/run.sh

```diff
@@ -178,6 +178,7 @@ function clone_submodules
             contrib/replxx
             contrib/wyhash
             contrib/eigen
+            contrib/nats-io
```

Comment:
let's remove it, it is not needed for fast test and it should be able to build even if we remove this.